packages/cli/src/utils/jsDocUtils.ts

```diff
@@ -18,6 +18,18 @@ export function getJSDocComment(node: ts.Node, tagName: string) {
   return tags[0].comment;
 }
 
+export function getJSDocComments(node: ts.Node, tagName: string) {
```

Comment:
This seems like a generalization of `getJSDocComment`, let's expose `getJSDocComment` as a wrapper of this new method that returns only the first element.